docs/UserGuide.md

```diff
@@ -164,7 +165,7 @@ Format: `remove INDEX [p/[PHONE]] [p/[GITHUBPROFILE]] [e/[EMAIL]] [a/[ADDRESS]]
 * At least one of the optional fields must be provided.
 * Removes the corresponding field value(s) matching the `KEYWORD`.
   * e.g. `l/Java` will remove `Java` from the languages of the contact.
-* Clears the field value(s) when no keyword is provided for the field.
+* Clears the field value(s) when no keyword is provided for the field even if the field is empty.
```

Comment:
Perhaps consider extracting the added section as a tip or note instead; e.g. "`remove` will remove a field successfully even if it is empty" or something